@types/graphql-query-complexity/index.d.ts

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
I think that this might become obsolete soon 😕 
https://github.com/ivome/graphql-query-complexity/commit/f7713b234472bcb14c63706f4cdee3ca882117cb
We should just wait with this PR for publishing that package with types.